docs/FAQ.md

```diff
@@ -276,14 +276,15 @@ Theoretically yes, the future is not really projectable beyond 80 years. However
 
 ### What type of wallet do you offer? Is it planned to offer hardware wallets?
 
-We currently have a desktop [wallet](https://github.com/alephium/desktop-wallet/releases/latest) and the team is working on mobile and [web extension wallets](https://github.com/alephium/extension-wallet/releases/latest). We want to offer the possibility to store ALPH on hardware wallets. We are currently pursuing being supported on Ledger. Please note that it is a lengthy process and won't be completed overnight.
+We currently have a [desktop wallet](https://github.com/alephium/desktop-wallet/releases/latest) and a [web extension wallet](https://github.com/alephium/extension-wallet/releases/latest). The team is actively working on a [mobile wallet](https://github.com/alephium/mobile-wallet). We want to offer the possibility to store ALPH on hardware wallets. We are currently pursuing being supported on Ledger. Please note that it is a lengthy process and won't be completed overnight.
```

Comment:
Alephium currently offers: 
- a [desktop wallet](https://github.com/alephium/desktop-wallet/releases/latest) 
- a [web extension wallet](https://github.com/alephium/extension-wallet/releases/latest) available on [Chrome] (https://chrome.google.com/webstore/detail/alephium-extension-wallet/gdokollfhmnbfckbobkdbakhilldkhcj) and [Firefox] (https://addons.mozilla.org/en-US/firefox/addon/alephiumextensionwallet/)

A [mobile wallet](https://github.com/alephium/mobile-wallet) is currently being worked on. 

Having Alephium available on hardware wallets is an important priority for the project. Ledger support is currently being worked on. Please note that it is a lengthy process with multiple stages that will take time to complete.